package.json

```diff
@@ -86,6 +86,7 @@
     "lodash": "4.17.21",
     "longjohn": "0.2.12",
     "marked": "2.0.1",
+    "mathjs": "^14.0.1",
```

Comment:
we need this for the `mean` function to calculate mean of multiple vectors. It brings with it some dependancies - I think we can implement our own `mean`. But at the same time - this code is expected to be short-lived - such computations will live in Gravity